### ISSUE TITLE
computeInfoVal2IFC: norm

### ISSUE THREAD

**Issue by matschmitz** [rdotsch/rcicr#96]
I think `norm(x, type = "f")` should be used (instead of `norm(x)`, see [discussion](https://stackoverflow.com/questions/10933945/how-to-calculate-the-euclidean-norm-of-a-vector-in-r)) in `computeInfoVal2IFC()`  to compute the "vector length" which has been defined as: "the square-root of the sum-of-squares over all pixel of the CIs" (p. 6, [Brinkman et al., 2018](https://psyarxiv.com/e5h8y)). 

E.g.:
```
> x <- matrix(1:3)
> sqrt(sum(x**2))
[1] 3.741657
> norm(x)
[1] 6
> norm(x, type = "f")
[1] 3.741657
```

Also, the denominator from `infoVal <- (cinorm - ref_median)/ref_mad` does not include the scaling factor `k = 1.4826` (p. 7).

**Comment by rdotsch** [comment]
Thank you for taking the time to look at the function and to report the issues. Both issues need to be fixed. 

You are more than welcome to submit a pull request with a fix for these, if you have the time.

**Comment by matschmitz** [comment]
Can't find the `computeInfoVal2IFC()` function in the repository. Could you tell me where it is located so I can make the pull request?

**Comment by rdotsch** [comment]
It's in the development branch, [here](https://github.com/rdotsch/rcicr/blob/development/R/computeInfoVal2IFC.R).

**Comment by rdotsch** [comment]
Fixed here #97. Thank you for contributing!
